src/dataSources/cloudFirestore/partner.js

```diff
@@ -1,20 +1,112 @@
 import debug from 'debug';
+import * as Sentry from '@sentry/node';
+import { dataSources, utility } from '@thatconference/api';
 
 const dlog = debug('that:api:partners:datasources:firebase:partner');
+const slugStore = dataSources.cloudFirestore.slug;
+const partnerDateForge = utility.firestoreDateForge.partners;
+
+function scrubPartner({ partner, isNew, user }) {
```

Comment:
ugh... me no like where this is headed.. there has to be a better way to validate this.

Reply:
What if we introuduced YUP to the mix here to validate things - https://github.com/jquense/yup I recognize it's accomplishing the same, but it could add some structure to things.

Reply:
Not sure I understand what you mean by validate this? What exactly are you seeing being validated? The URL's are in a scalar URL format and need to be changed to a string before being written to the database.